### DISCUSSION TITLE
We need to update the design contribution guidelines to be comprehensive

### DISCUSSION THREAD

**Discussion by itsreallynotadam** [cengage/react-magma#72] [General]
@orion-cengage , can we make sure the design contribution guidelines are clearly articulated and included as a part of the docs. We want to make sure the standards we've accounted for are available and reference able.

**Comment by orion-cengage** [comment]
@itsreallynotadam , How would you feel about addressing contribution guidelines on a single page, and putting that page at the same level as the Introduction?  I feel we can handle all perspectives (designer, dev, or both) most efficiently from a single page.

**Reply by itsreallynotadam** [reply]
I think that's fine, assuming the root level standards are shared (design/dev) and then have a specific documentation for accepting components via sketch akin to the contributor docs for the component API?

**Reply by joasmi723** [reply]
The development contribution guidelines are kind of long, and have information that wouldn't relevant to most designers.  I'd assume the same would be true for the design contribution guidelines.  But I agree with Adam,   I could see having root-level standards info and then linking off to more in-depth guidelines, depending on how the content ended up.

**Reply by orion-cengage** [reply]
Thanks to you both for the feedback.  I'm going to attempt to combine them and see if it's possible.  After reading through the dev contribution guidelines, there's a bunch of stuff in there that feels appropriate to even non-devs, like the sections about communication, accessibility, and collaboration.  I'll report back when I have something for you to review.

**Comment by joasmi723** [comment]
It also occurred to me that we're going to need to update the developer guidelines to describe the new use of Code Sandbox and Storybook.  So the dev guidelines might get a little longer -- just a heads up.  :)

**Reply by orion-cengage** [reply]
That's definitely good to know.  I would love to see an updated set of documentation from whoever is going to be writing that, so I can see how it will impact what I've done so far.
